crypto/src/main/scala/org/bitcoins/crypto/Sign.scala

```diff
@@ -24,28 +24,85 @@ trait Sign {
   def signFuture(bytes: ByteVector): Future[ECDigitalSignature] =
     signFunction(bytes)
 
+  def signWithEntropyFunction: (
+      ByteVector,
+      ByteVector) => Future[ECDigitalSignature]
+
+  def signWithEntropyFuture(
+      bytes: ByteVector,
+      entropy: ByteVector): Future[ECDigitalSignature] =
+    signWithEntropyFunction(bytes, entropy)
+
+  private def signLowRFuture(bytes: ByteVector, startAt: Long)(implicit
+      ec: ExecutionContext): Future[ECDigitalSignature] = {
+    val startBytes = ByteVector.fromLong(startAt).padLeft(32)
+    val startPlusOneBytes = ByteVector.fromLong(startAt + 1).padLeft(32)
+
+    val sig1F: Future[ECDigitalSignature] =
```

Comment:
Hmm, so it appears the low r signing stuff won't play well with rate limiting schemes in a key manager? If I understand correctly the idea is just to keep generating signatures until you get one below 70 bytes?

Reply:
Perhaps we should keep this simple for now and just generate one signature and check it and then recurse rather than generating two signatures in parallel?